### PR TITLE
feat components

### PR DESCRIPTION
**Pull request by DenislavK8**

### CODE REVIEW

File: src/Form.css [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  background-color: rgb(191, 231, 139);
+}
+
+.hiddenInput {
+  display: none;
+}
+
+.select {
+  background-color: rgb(191, 231, 139);
+}
```
Comment (preslavnolimit) [inline src/Form.css:21]: space.

Comment (DenislavK8) [reply src/Form.css:21]: done

File: src/Form.css [hunk 2]
```diff
@@ -0,0 +1,31 @@
+.form {
+  margin: 0px 500px 0px 500px;
+  background-color: rgb(240, 124, 95);
+  border: 1px solid black;
+}
+
+.textarea {
+  background-color: rgb(191, 231, 139);
+}
```
Comment (preslavnolimit) [inline src/Form.css:9]: add space.

Comment (DenislavK8) [reply src/Form.css:9]: done

**Pull request merged**